cognite/client/data_classes/assets.py

```diff
@@ -638,14 +638,14 @@ def parent_sort_fn(asset: Asset) -> str:
         )
         return mapping
 
-    def count_subtree(self, mapping: Dict[Optional[str], List[Asset]]) -> Dict[str, int]:
+    def count_subtree(self, mapping: Dict[Optional[str], List[Asset]]) -> Dict[Optional[str], int]:
```

Comment:
Please revert type annotation changes for `count_subtree`. Check line 666, the mapping is purely str->str and hopefully mypy will understand that in a future version. Use —no-verify to skip pre-commit if you’re not on 3.8 😊

Reply:
str->int * 😅

Reply:
Done. Sorry for the late response. Was a crazy week :)